codec-http2/src/test/java/io/netty/handler/codec/http2/DataCompressionHttp2Test.java

```diff
@@ -89,14 +89,14 @@
     public void setup() throws InterruptedException, Http2Exception {
         MockitoAnnotations.initMocks(this);
         doAnswer(invocation -> {
-            if (invocation.getArgument(4)) {
+            if ((boolean) invocation.getArgument(4)) {
```

Comment:
these changes in tests are unrelated to this PR

Reply:
As I mentioned in PR comment.